### DISCUSSION TITLE
Add company info to profile mouseover

### DISCUSSION THREAD

**Discussion by eddiezane** [community/community#6786] [Other Feature Feedback, Questions, & Ideas]
I help maintain a large open source project with different corporate involvement. Often times knowing the company a user works for can help provide context (use case) to feature requests and bug reports. Having this information displayed (near where a user's location is currently shown) would save time from having to open profiles in a new tab.

![image](https://user-images.githubusercontent.com/959573/138973551-fb9330d2-0ea2-4e85-ba7f-1028e2f78692.png)

**Comment by alimttth** [comment]
as for unlimited links, you can use the default README on your landing page to include all of your links, you can even make it beautiful by making a link icon instead of the actual link  this is what it looks like
